saleor/graphql/core/mutations.py

```diff
@@ -596,12 +602,29 @@ def mutate(cls, root, info, **kwargs):
             result = super().mutate(root, info, **kwargs)
         except JSONWebTokenError as e:
             return CreateToken(errors=[Error(message=str(e))])
+        except ValidationError as e:
+            errors, account_errors = cls.handle_errors(e)
+            return CreateToken(errors=errors, account_errors=account_errors)
         else:
             return result
 
+    @classmethod
+    def handle_errors(cls, error: ValidationError, **extra):
+        errors = validation_error_to_error_type(error)
+        return cls.handle_typed_errors(errors)
+
+    @classmethod
+    def handle_typed_errors(cls, errors: list):
```

Comment:
We have `handle_typed_errors` in `SetPassword` mutaton. 
`SetPassword` inherrit from `CreateToken` mutation. 
We can remove `handle_typed_errors` from `SetPassword`

Reply:
Done